components/02-molecules/banner/cta/cta-banner.twig

```diff
@@ -0,0 +1,60 @@
+{#
+ # Available Props:
+ # - banner__content__layout: bottom (default), left, or right
+ #
+ # Available Variables:
+ # - banner__heading
+ # - banner__snippet
+ #
+ # Available Blocks:
+ # - banner__image
+ #}
+
+{% set banner__base_class = 'cta-banner' %}
+{% set banner__content__background = banner__content__background|default('gray-800') %}
+
+{% set banner__attributes = {
+  'data-banner-content-layout': banner__content__layout|default('bottom'),
+  class: bem(banner__base_class),
+} %}
+
+<div {{ add_attributes(banner__attributes) }}>
+  <div {{ bem('content-wrapper', [], banner__base_class) }}>
+    <div {{ bem('image', [], banner__base_class) }}>
+      {% block banner__image %}
+        {% include "@atoms/images/image/_responsive-image.twig" %}
+      {% endblock %}
+    </div>
+    <div {{ bem('content', [], banner__base_class) }} data-component-theme={{ banner__content__background }}>
+      <div {{ bem('text', [], banner__base_class) }}>
+        {% include "@atoms/typography/headings/heading.twig" with {
+          heading__level: '1',
```

Comment:
This is the only line that is sticking out to me as something we might want to change. Currently, this is statically setting the heading to be an `h1` which is what the linked design calls for. However, if this Banner CTA is used or repurposed for any other type of instance (available via a components field, repurposed as a carousel slide) it will cause errors. I wonder if there should be a `banner__heading_level` variable that defaults to `1` but can be easily updated to any other number during implementation.

But it does meet the requirements of the current design so feel free to say "will refactor when appropriate"

Reply:
Yeah, I originally had it as an h2 actually, but found out that this will REPLACE the page title, if used. So the current expectation is that this IS the page title. There are no current designs to suggest this will be used in a carousel, or anywhere else on the page.

Jim and I discussed the possibility that these could be used in the content area at a later time. But if we do that, there will be other (Drupal) refactoring required anyway, so we'll handle the frontend portion at that time, if the need comes up.